basic.css

```diff
@@ -0,0 +1,87 @@
+html {
+  height: 100%;
+}
+
+body {
+  min-height: 100%;
+  margin: 0%;
+}
+
+.toolbar {
+  background-color: #3c3a39;
+  margin: 0%;
+  padding: 12px 24px;
+  height: fit-content;
+}
+
+.dropdown {
+  margin-left: auto;
+}
+
+.fontFam {
+  font-family: 'Roboto', sans-serif;
+}
+
+.textcolor {
+  color: #fff;
+}
```

Comment:
## [OPTIONAL]
- [ ] Kindly I would suggest you remove the underline on the text logo. Hint :point_down: 

```css
.textcolor {
  color: #fff;
  text-decoration: none;
}
```

Reply:
Thanks, @andersonlebon for the valuable review. Fix this too.